src/utilities/Blacklist.ts

```diff
@@ -0,0 +1,10 @@
+import DEFAULT_BLACKLIST from './../lang/en.json';
```

Comment:
the en.json represents localization not DEFAULT_BLACKLIST

Reply:
I changed to another name. Please help me check it